### PR TITLE
Enable fiat-crypto

### PR DESCRIPTION
**Pull request by JasonGross**
This closes [bug 5607](https://coq.inria.fr/bugs/show_bug.cgi?id=5607).

PR #220 put the exports in the wrong compat files, presumably because it
was originally targeted to version 8.6, and this wasn't updated when it
was retargeted to version 8.7.

### CODE REVIEW

File: dev/ci/ci-user-overlay.sh
Comment (Zimmi48) [inline dev/ci/ci-user-overlay.sh]: This is not something you should care about. Fix `ci-basic-overlay.sh` instead.

**Review by Zimmi48 — CHANGES_REQUESTED** [review]
LGTM modulo the remark about fixing overlays.

File: dev/ci/ci-user-overlay.sh
Comment (JasonGross) [inline dev/ci/ci-user-overlay.sh]: Oh, oops, I reverted the wrong commit

Comment (JasonGross) [inline dev/ci/ci-user-overlay.sh]: Should I edit just the basic-overlay, or also this one?

Comment (Zimmi48) [inline dev/ci/ci-user-overlay.sh]: This one you can leave unchanged. But it doesn't matter actually.

**Review by Zimmi48 — APPROVED** [review]
To have this PR properly tested by Travis, you should probably add to this PR the commit from #792.

**Comment by JasonGross** [comment]
The overlay reverting depends on #792; should I cherry-pick the relevant bits into this PR?

**Comment by JasonGross** [comment]
Rebased, now on top of commits cherry-picked from #792

**Comment by maximedenes** [comment]
@JasonGross Could you rebase now that #792 has been merged? The history is going to look a bit messy otherwise. Thanks!

**Comment by JasonGross** [comment]
I've rebased to drop the conflicting change to `ci-user-overlay`, but, @maximedenes , #792 was on v8.6, and this is on trunk, so rebasing won't help there until you (or someone) merges v8.6 into trunk again.

**Pull request merged**
